### PR TITLE
Eth event timeout

### PR DESCRIPTION
**Pull request by yito88**
https://github.com/anoma/namada/issues/693
https://github.com/anoma/namada/issues/181

### CODE REVIEW

**Review by batconjurer — COMMENTED** [review]
Just a variable name change. Otherwise looks good.

File: ethereum_bridge/src/protocol/transactions/ethereum_events/mod.rs [hunk 1]
```diff
@@ -157,19 +163,99 @@ where
...
     Ok((changed, confirmed))
 }
 
+fn timeout_events<D, H>(storage: &mut Storage<D, H>) -> Result<ChangedKeys>
+where
+    D: 'static + DB + for<'iter> DBIter<'iter> + Sync,
+    H: 'static + StorageHasher + Sync,
+{
+    let mut changed = ChangedKeys::new();
+    for keys in get_timeout_events(storage) {
```
Comment (sug0) [inline ethereum_bridge/src/protocol/transactions/ethereum_events/mod.rs:186]: nit: rename to `get_timed_out_eth_events()`?

File: ethereum_bridge/src/storage/vote_tallies.rs [hunk 2]
```diff
@@ -69,6 +73,13 @@ impl<T> Keys<T> {
             .push(&VOTING_POWER_KEY_SEGMENT.to_owned())
             .expect("should always be able to construct this key")
     }
+
+    /// Get the `epoch` key - there should be a `Epoch` stored here.
```
Comment (sug0) [inline ethereum_bridge/src/storage/vote_tallies.rs:77]:
```suggestion
    /// Get the `epoch` key - there should be an [`Epoch`] stored here.
```

**Review by sug0 — CHANGES_REQUESTED** [review]
nits

**Review by batconjurer — APPROVED** [review]

**Pull request merged**